engines/tetraedge/te/te_font3.h

```diff
@@ -86,7 +86,9 @@ class TeFont3 : public TeResource {
 private:
 	void init();
 	Graphics::Font *getAtSize(uint size);
+	Common::CodePage codePage();
```

Comment:
as a nit, I guess this should be static or at least const

Reply:
Thank you, I think it can't be static because it uses g_engine.
const on the return type is a warning (Common::CodePage is enum)
added const that marks that the instance does not change, but it might be relevant to more methods here.

please note that this methof is currently used once in the constructor.

Reply:
I could have been more clear - yep, setting the function const is what I meant, which means it won't change the contents of the object, not return const.

I think it could also be `static` which in c++ has a bit of a different meaning to Java in that it just means "does not depend on which instance of this class it's called on".

Since `g_engine` is a global rather than a class member it's also be fine to make this function static, but I haven't been super consistent about that myself so `const` is also fine as it's not performance-critical code.

Thanks again for the fix!